l10n_th_account_tax_filing/models/account_move.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2025 Ecosoft Co., Ltd. (http://ecosoft.co.th)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import fields, models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    tax_filing_id = fields.Many2one(
```

Comment:
```suggestion
    tax_filing_id = fields.Many2one(
        readonly=True,
        index=True,
```

- Move this field to `account.move.line`
- Change name to `filing_line_id` or `tax_filing_line_id`
- add readonly and index